roles/redhat_satellite6_capsule_lb/tasks/main.yml

```diff
@@ -0,0 +1,92 @@
+---
+# tasks file for satellite_lb
+
+- name: Generate LB Certs 1
+  when: inventory_hostname in groups['satellite']
```

Comment:
@Smithe2413 what happens if they have more then one host int heir statellite group? woudln't it be better to put the satellite master tasks into a seperate task file and then include that task fiel only for the first host listed in `groups['satellite']`